### DISCUSSION TITLE
Repo Creation

### DISCUSSION THREAD

**Discussion by AliHJMM** [community/community#134721] [New to GitHub] [answered]
How to create github repo?

**Comment by Mohamed-Alasfoor** [comment, marked as answer]
go to repositories and press new repo , name it and create it

**Comment by Mainasploit** [comment]
Hi @AliHJMM, 
Here's a step-by-step guide you can use to create a GitHub repository:

### Step-by-Step Guide to Create a GitHub Repository

1. **Go to GitHub**:
   - Open your web browser and go to [GitHub](https://github.com).
   - Make sure you're logged in to your account.

2. **Create a New Repository**:
   - In the top-right corner of any GitHub page, click the "+" icon, then click "New repository".

     ![New Repository](https://docs.github.com/assets/images/help/repository/repo-create.png)

3. **Fill in Repository Details**:
   - **Repository Name**: Enter a name for your repository. For example, "my-new-repo".
   - **Description**: Optionally, add a description for your repository, like "This is my first repository".
   - **Public/Private**: Choose whether the repository should be public (visible to everyone) or private (visible only to you and people you invite).
   - **Initialize with a README**: Check this box to add a README file. This file is useful for providing information about your project.
   - **Add .gitignore**: Optionally, select a .gitignore template suited for your project to specify which files should be ignored by Git.
   - **Choose a license**: Optionally, select a license for your project to determine how others can use your code.

     ![Repository Details](https://docs.github.com/assets/images/help/repository/create-repository-name.png)

4. **Create the Repository**:
   - After filling in the details, click the "Create repository" button at the bottom. </br> 
 **VIOLA! THERE'S YOUR NEWLY CREATED REPORSITORY**.
